velox/exec/tests/WindowTest.cpp

```diff
@@ -160,14 +160,28 @@ TEST_F(WindowTest, spillUnsupported) {
 TEST_F(WindowTest, rowBasedStreamingWindowOOM) {
   const vector_size_t size = 1'000'000;
   auto data = makeRowVector(
-      {"d", "p", "s"},
+      {"d", "p", "s", "d4", "d5", "d6", "d7", "d8","d9", "d10"},
       {
           // Payload.
           makeFlatVector<int64_t>(size, [](auto row) { return row; }),
           // Partition key.
           makeFlatVector<int16_t>(size, [](auto row) { return row; }),
           // Sorting key.
           makeFlatVector<int32_t>(size, [](auto row) { return row; }),
+          // Payload.
+          makeFlatVector<int64_t>(size, [](auto row) { return row; }),
+           // Payload.
+           makeFlatVector<int64_t>(size, [](auto row) { return row; }),
+           // Partition key.
```

Comment:
Don't follow the use of // Partition key and //Sorting key for these columns...Are they used in PARTITION BY and ORDER BY clauses anywhere ?

Reply:
@aditi-pandit Just for local test. And I have removed this code.